### PR TITLE
add wbf integration

### PR DESCRIPTION
**Pull request by madnai**

### CODE REVIEW

**Comment by kroitor** [comment]
Hey @madnai ! Thank you for your PRs! Pardon for the long wait, I'll get to them asap!

File: js/wbf.js [hunk 1]
```diff
@@ -0,0 +1,616 @@
...
+                query = this.urlencode ({
+                    'api_key': apiKey,
+                    'time': timestamp,
+                    'sign': signature,
+                });
+                if (Object.keys (params).length) {
+                    url += '&' + query;
+                } else {
+                    url += '?' + query;
+                }
```
Comment (lesterkimkraken) [inline js/wbf.js:595]:
```suggestion
                symbol = Object.keys(params).length ? '&' : '?';
                url += `{symbol}{query}`;
```

Comment (kroitor) [reply js/wbf.js:595]: @lesterkimkraken that will not transpile to other languages: 

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#multilanguage-support
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

Comment (lesterkimkraken) [reply js/wbf.js:595]: Ah, thanks for the background.

**Comment by ttodua** [comment]
dead exchange

**Pull request closed without merging**